src/main/java/edu/wisc/my/ltiproxy/LTIParameters.java

```diff
@@ -0,0 +1,26 @@
+package edu.wisc.my.ltiproxy;
+
+import java.util.Collections;
+import java.util.Map;
+
+/**
+ *
+ * @author sibley
```

Comment:
I don't favor `@author` annotations in shared-ownership codebases, but I recognize it's a matter of style and reasonable developers can differ on this point.